apps/api.planx.uk/modules/send/fme/index.ts

```diff
@@ -0,0 +1,81 @@
+import { subDays } from "date-fns";
+import type { NextFunction, Request, Response } from "express";
+import { gql } from "graphql-request";
+import { $api } from "../../../client/index.js";
+
+export async function getSubmissionsController(
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) {
+  const localAuthority = req.params?.localAuthority;
+  if (!localAuthority) {
+    return next({
+      status: 400,
+      message: "Missing values required to access submissions",
+    });
+  }
```

Comment:
I think it's worth setting up a simple validation schema for this. Right now it's just one param but we're pretty sure filtering will be an upcoming requirement here.

Reply:
Yep very fair, validation schema now added :heavy_plus_sign: